<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="6" total="7" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="400653" started-at="2014-02-10T09:31:36Z" finished-at="2014-02-10T09:38:16Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="400653" started-at="2014-02-10T09:31:36Z" finished-at="2014-02-10T09:38:16Z">
      <class name="hub.test.SmokeTest">
        <test-method status="PASS" signature="Startup(java.lang.String)[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="Startup" is-config="true" duration-ms="30610" started-at="2014-02-10T17:31:36Z" finished-at="2014-02-10T17:32:06Z">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Startup -->
        <test-method status="PASS" signature="testSmokeTest(java.lang.Class, [Ljava.lang.String;)[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="testSmokeTest" duration-ms="70648" started-at="2014-02-10T17:32:07Z" description="Smoke Test" data-provider="Data-Provider-Function" finished-at="2014-02-10T17:33:17Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[class java.lang.String]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.String;@688d9b8]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSmokeTest -->
        <test-method status="PASS" signature="reinitializeBrowser()[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="reinitializeBrowser" is-config="true" duration-ms="2259" started-at="2014-02-10T17:33:17Z" finished-at="2014-02-10T17:33:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reinitializeBrowser -->
        <test-method status="FAIL" signature="testSmokeTest(java.lang.Class, [Ljava.lang.String;)[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="testSmokeTest" duration-ms="70671" started-at="2014-02-10T17:33:19Z" description="Smoke Test" data-provider="Data-Provider-Function" finished-at="2014-02-10T17:34:30Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[class java.lang.String]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.String;@398ce07d]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Exception was thrown]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Exception was thrown
	at org.testng.Assert.fail(Assert.java:94)
	at hub.test.SmokeTest.testSmokeTest(SmokeTest.java:86)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSmokeTest -->
        <test-method status="PASS" signature="reinitializeBrowser()[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="reinitializeBrowser" is-config="true" duration-ms="2272" started-at="2014-02-10T17:34:30Z" finished-at="2014-02-10T17:34:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reinitializeBrowser -->
        <test-method status="FAIL" signature="testSmokeTest(java.lang.Class, [Ljava.lang.String;)[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="testSmokeTest" duration-ms="88544" started-at="2014-02-10T17:34:32Z" description="Smoke Test" data-provider="Data-Provider-Function" finished-at="2014-02-10T17:36:01Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[class java.lang.String]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.String;@4fe64c0f]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Exception was thrown]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Exception was thrown
	at org.testng.Assert.fail(Assert.java:94)
	at hub.test.SmokeTest.testSmokeTest(SmokeTest.java:86)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSmokeTest -->
        <test-method status="PASS" signature="reinitializeBrowser()[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="reinitializeBrowser" is-config="true" duration-ms="2081" started-at="2014-02-10T17:36:01Z" finished-at="2014-02-10T17:36:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reinitializeBrowser -->
        <test-method status="FAIL" signature="testSmokeTest(java.lang.Class, [Ljava.lang.String;)[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="testSmokeTest" duration-ms="51133" started-at="2014-02-10T17:36:03Z" description="Smoke Test" data-provider="Data-Provider-Function" finished-at="2014-02-10T17:36:54Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[class java.lang.String]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.String;@188b95c4]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Exception was thrown]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Exception was thrown
	at org.testng.Assert.fail(Assert.java:94)
	at hub.test.SmokeTest.testSmokeTest(SmokeTest.java:86)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSmokeTest -->
        <test-method status="PASS" signature="reinitializeBrowser()[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="reinitializeBrowser" is-config="true" duration-ms="2164" started-at="2014-02-10T17:36:54Z" finished-at="2014-02-10T17:36:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reinitializeBrowser -->
        <test-method status="FAIL" signature="testSmokeTest(java.lang.Class, [Ljava.lang.String;)[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="testSmokeTest" duration-ms="56262" started-at="2014-02-10T17:36:56Z" description="Smoke Test" data-provider="Data-Provider-Function" finished-at="2014-02-10T17:37:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[class java.lang.String]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.String;@3068ece8]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Exception was thrown]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Exception was thrown
	at org.testng.Assert.fail(Assert.java:94)
	at hub.test.SmokeTest.testSmokeTest(SmokeTest.java:86)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSmokeTest -->
        <test-method status="PASS" signature="reinitializeBrowser()[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="reinitializeBrowser" is-config="true" duration-ms="2511" started-at="2014-02-10T17:37:53Z" finished-at="2014-02-10T17:37:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reinitializeBrowser -->
        <test-method status="FAIL" signature="testSmokeTest(java.lang.Class, [Ljava.lang.String;)[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="testSmokeTest" duration-ms="16414" started-at="2014-02-10T17:37:55Z" description="Smoke Test" data-provider="Data-Provider-Function" finished-at="2014-02-10T17:38:12Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[class java.lang.String]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.String;@38998c2c]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Exception was thrown]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Exception was thrown
	at org.testng.Assert.fail(Assert.java:94)
	at hub.test.SmokeTest.testSmokeTest(SmokeTest.java:86)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSmokeTest -->
        <test-method status="PASS" signature="reinitializeBrowser()[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="reinitializeBrowser" is-config="true" duration-ms="1768" started-at="2014-02-10T17:38:12Z" finished-at="2014-02-10T17:38:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reinitializeBrowser -->
        <test-method status="FAIL" signature="testSmokeTest(java.lang.Class, [Ljava.lang.String;)[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="testSmokeTest" duration-ms="2" started-at="2014-02-10T17:38:13Z" description="Smoke Test" data-provider="Data-Provider-Function" finished-at="2014-02-10T17:38:13Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[class java.lang.String]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[[Ljava.lang.String;@68c1b9bb]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[27]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 27
	at hub.test.SmokeTest.testSmokeTest(SmokeTest.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testSmokeTest -->
        <test-method status="PASS" signature="reinitializeBrowser()[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="reinitializeBrowser" is-config="true" duration-ms="2818" started-at="2014-02-10T17:38:13Z" finished-at="2014-02-10T17:38:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- reinitializeBrowser -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:hub.test.SmokeTest@1c06d198]" name="teardown" is-config="true" duration-ms="181" started-at="2014-02-10T17:38:16Z" finished-at="2014-02-10T17:38:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- hub.test.SmokeTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
